tests/node_converters/pad_test.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import onnx
+import pytest
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+def _test_pad(
+        input_array: np.ndarray,
+        opset_version: int,
+        **kwargs,
+) -> None:
+
+    test_inputs = {
+        'x': input_array,
+    }
+
+    if opset_version != 2:
+        test_inputs['pads'] = np.array(kwargs.pop('pads'), dtype=np.int64)
+
+    node = onnx.helper.make_node(
+        'Pad',
+        inputs=list(test_inputs),
+        outputs=['y'],
+        **kwargs,
+    )
+
+    model = make_model_from_nodes(
+        nodes=node,
+        initializers={},
+        inputs_example=test_inputs,
+        opset_version=opset_version,
+    )
+    check_onnx_model(model, test_inputs)
+
+
+@pytest.mark.parametrize(
+    'pads,mode',
+    (
+        ([0,0,0,0,0,0,0,0], 'constant'),
+        ([0,1,1,1,1,0,0,0], 'constant'),
+        ([0,2,0,2,0,2,0,2], 'constant'),
+        ([1,2,4,2,5,4,4,2], 'constant'),
+        ([0,0,0,0,0,0,0,0], 'edge'),
+        ([0,0,2,3,0,0,2,3], 'edge'),
+        ([0,0,0,0,0,0,0,0], 'reflect'),
+        ([0,0,2,1,0,0,2,1], 'reflect'),
+    )
+)
+@pytest.mark.parametrize(
```

Comment:
Just 1 line, you dont need 3 lines here.

And you should add tests for 3d and 5d tensors.

Reply:
Done